outlines/react.md

```diff
@@ -0,0 +1,34 @@
+# React
+
+1. Introduction to React
+  1. Background: developed by Facebook, used widely in production, vibrant ecosystem
+  2. Installing and using React in Meteor 1.3
+  3. A simple example
+
+2. JSX in depth
+  1. What JSX really is (helping people understand it)
+  2. JSX workflow (transpiling - we've got it covered for you!)
+  3. JavaScript expressions inside JSX
+  4. Conditional and list rendering with JSX
+  5. Gotchas
+
+3. Components in depth
+  1. State
```

Comment:
I think we will have to touch upon state first in the first simple example, just to show something that can handle interactivity, so explaining state first is a bit more natural. Of course we can be careful about the phrasing and mention the danger of abusing local state (which will probably be discussed more in the container vs. presentational component part below)